go/libkb/errors.go

```diff
@@ -1358,6 +1358,15 @@ func (e IdentifyFailedError) Error() string {
 
 //=============================================================================
 
+type IdentifiesFailedError struct {
+}
+
+func (e IdentifiesFailedError) Error() string {
+	return fmt.Sprintf("one or more identifies failed")
```

Comment:
just `return "one or more..."`

Reply:
ok, why?

Reply:
it's just a string, so just return it...there isn't any formatting of it going on.

Reply:
oh, heh, I thought you meant make it an ellipsis